torch/ao/quantization/fx/convert.py

```diff
@@ -533,11 +533,15 @@ def convert_custom_module(
         assert isinstance(prev_node, Node), \
             f"Expecting the argument for custom module node to be a Node, but got {prev_node}"
         if prev_node.op == "call_method" and prev_node.target == "dequantize":
-            assert len(prev_node.users) == 1, "dequantize node before custom module is used "
-            "multiple times, this is currently not supported yet, but it can be "
-            "supported by duplicating the dequantize nodes in these cases"
-            prev_node.replace_all_uses_with(prev_node.args[0])
-            graph.erase_node(prev_node)
+            # change the connection for custom module, we'll change the input
+            # of custom module node to quantize node:
+            # Before: quantize - dequantize - custom - module
+            # After: quantize - custom - module
+            #              \ - dequantize
+            #                  (may be dangling depending on how many users it has before)
```

Comment:
can we just do the check here to clean it up instead of relying on DCE?  This way the intermediate state will be easier to understand, for any developers that need to work on the intermediate state (to fix a bug, to develop a new feature, etc)

Reply:
I see, yeah I think we can do that as well